content/vault/v1.21.x (rc)/content/api-docs/system/internal-counters.mdx

```diff
@@ -1004,6 +1004,260 @@ $ curl \
     http://127.0.0.1:8200/v1/sys/internal/counters/activity?end_time=2020-06-30T00%3A00%3A00Z&start_time=2020-06-01T00%3A00%3A00Z&limit_namespaces=2
 ```
 
+## Cumulative namespace client count
+
+@include 'alerts/enterprise-only.mdx'
+
+@include 'alerts/tech-preview.mdx'
+
+The cumulative client count endpoint returns, for each namespace, the total number of clients in that namespace and all its descendant namespaces.
+
+You can call the cumulative client count endpoint from any namespace. However, to query client counts for deleted namespaces, you must call the endpoint from `root` since the original namespace path is unknown. Client counts in deleted namespaces also do not contribute to the cumulative counts of any namespace other than `root`.
+
+<Tip title="Related endpoint">
+
+For client counts isolated to each namespace, grouped by month, or to see
+mount-level attribution, use `sys/internal/counters/activity`.
+
+</Tip>
+
+Regardless of the values provided for `start_time` and `end_time`, Vault can
+only return totals for whole billing periods. If the requested time parameters
+do not align with a complete billing period, Vault adjusts the request to the
+full billing period associated with the provided times.
+
+The endpoint response always includes the actual time period covered in addition
+to the client counts. If Vault adjusts the query period, the result also
+includes a warning about the adjusted query times.
+
+If `start_time` and `end_time` do not fall within the same billing period, Vault
+uses the billing period associated with `start_time`.
+
+<Warning>
+
+If the number of namespaces in the response approaches or exceeds the
+[default maximum](/vault/docs/internals/limits#namespace-limits), the request
+may take several seconds to complete. To prevent impacting the performance of
+your Vault cluster, avoid making multiple requests to the cumulative count
+endpoint at once. Consider setting a [rate limit quota](/vault/docs/configuration/create-rate-limit-quota)
+on the endpoint to prevent excessive requests.
+
+</Warning>
+
+| Method | Path                                         |
+|:-------|:---------------------------------------------|
+| `GET`  | `/sys/internal/counters/activity/cumulative` |
+
+### Parameters
+
+- `start_time` `(string, optional)` - An RFC3339 timestamp or Unix epoch time
```

Comment:
Similarily, start_time will always be aligned to the start of a billing period

Reply:
Changed the wording to specify start of the billing period